packages/model-viewer/src/features/scene-graph.ts

```diff
@@ -283,6 +295,33 @@ export const SceneGraphMixin = <T extends Constructor<ModelViewerElementBase>>(
     [$onModelGraftMutation](_event: Event) {
       this[$needsRender]();
     }
+
+    /** @export */
+    async exportScene(options: {
+			binary: false,
+			trs: false,
+			onlyVisible: true,
+			truncateDrawRange: true,
+			embedImages: true,
```

Comment:
If this and binary are false, does our blob contain a bunch of files? Have you tried this?

Reply:
I tried, and if embedImages is false, no textures get exported.  What would be your expected scenario in such case?